src/components/expanded-state/asset/ChartExpandedState.js

```diff
@@ -187,24 +188,15 @@ export default function ChartExpandedState({ asset }) {
 
   const delayedDescriptions = useDelayedValueWithLayoutAnimation(data?.description?.replace(/\s+/g, ''));
 
-  const scrollableContentHeight = true;
   const { chart, chartType, color, fetchingCharts, updateChartType, initialChartDataLabels, showChart, throttledData } =
     useChartThrottledPoints({
       asset: assetWithPrice,
       heightWithChart: Math.min(
-        carouselHeight +
-          heightWithChart -
-          (!hasBalance && 68) +
-          additionalContentHeight +
-          (additionalContentHeight === 0 ? 0 : scrollableContentHeight),
+        carouselHeight + heightWithChart - (!hasBalance && 68) + additionalContentHeight + (additionalContentHeight === 0 ? 0 : true),
```

Comment:
I know this isn't yours originally but I'm having a hard time understanding this code. Specifically the purpose of the final conditional: `(additionalContentHeight === 0 ? 0 : true)`.

Using a boolean within `Math.min` treats `true` as `1` and `false` as `0`.
<img width="187" alt="Screenshot 2025-01-09 at 3 33 16 PM" src="https://github.com/user-attachments/assets/59d3d1af-2d1b-445b-aa33-0b795060e896" />

What is the significance of adding `1` when `additionalContentHeight` is `0`?